### ISSUE TITLE
508 issue, Identity of specific button is not announced by JAWS

### ISSUE THREAD

**Issue by nnarvekar** [jquense/react-big-calendar#2046]
#### What's the current behavior?
JAWS announces the event title in the calendar. For example it reads out the event as "Test Event”. But its expected to read "Test Event button to activate press spacebar.”

#### What's the expected behavior?
In addition to reading the title, the screen reader should read additional information like "to activate press spacebar"

**Comment by nnarvekar** [comment]
> @nnarvekar It is something that can be fixed. With direction. Do you know the proper ARIA attributes to use?

@cutterbl I dont know the ARIA attributes to use
